zwave-property.js

```diff
@@ -84,6 +84,9 @@ class ZWaveProperty extends Property {
         motion = true;
         break;
     }
+    if (motion === undefined) {
```

Comment:
This is safer:

```javascript
if (typeof motion === 'undefined') {
```

Ditto below.

Reply:
Hah, sorry Dave. (He asked me how to test for undefined in JavaScript).